PrincetonUSG/TigerSnatch
# Make Activity page and status indicator public; add Statistics to introJS tutorial

## Goal
- Allow non-Princeton individuals to view high-level statistics about TigerSnatch activity.
- Simplify logged-out navigation bar.

## Changes
- Logged-out users can now access the Activity page and see the status indicator. Subscriptions and Trades logs display placeholder messages since they require logging in.

<img width="1453" alt="image" src="https://user-images.githubusercontent.com/13815069/149648933-7ecb884f-5801-4a4f-a788-c11cd9448e31.png">

- The logged-out navbar has been simplified. It now displays the same items no matter which page a user is on (previously, it would display all pages except for the current page, resulting in the navbar links shifting around when heading to another page).

<img width="381" alt="image" src="https://user-images.githubusercontent.com/13815069/149648976-1e4fd4b4-04f3-42c5-81dd-b2e4bf1dabb0.png">

- The introJS tutorial now highlights the Statistics panel. Also, the localStorage key is reset so all users will experience the tutorial again.

<img width="264" alt="image" src="https://user-images.githubusercontent.com/13815069/149649817-22cd998a-3edd-4a88-b595-04301649bb74.png">

- The introJS tutorial is now un-skippable. Pressing escape and clicking away from the popup are now disabled, and the "x" in the upper right is gone.

## Testing
- Log out of TigerSnatch.
- Click on the links in the navbar and check that they all work properly.
- Check that the Activity page shows the Statistics panel, and shows placeholder messages for the Subscriptions and Trade logs.
- Check that the status indicator is properly populated with text.
- Log into TigerSnatch.
- Test the tutorial. Complete it and reload the page to check that it doesn't appear again.
